open-metadata-implementation/governance-servers/virtualization-services/readme.md

```diff
@@ -0,0 +1,82 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+# Virtualizer
+Virtualizer communicates with Information View OMAS and virtualization tool which is currently Gaian.
+
+Virtualizer has three main functions:
+1. listen to Information View OMAS Out topic(specified by property information-view-out-topic) and retrieve InformationViewEvent event (json structure);
+2. create Business Logical View(business terms are used as column names) and Technical Logical View(source table column names are used as view column names), containing only the columns with business terms assigned
+3. create Business View json file and Technical View json file, notify Information View OMAS through publishing on Information View OMAS In topic (specified by property information-view-in-topic).
+
+## OMAG server configuration
+
+Virtualizer is now running as a service for [OMAG Application Platform](../server-chassis). In this case, the service should be configured and initialized by the restful APIs provided by the platform.
+
+Here are the steps to run Vitualizer
+- Start Server Chassis
+- Configure the server URL with an HTTP **POST**
+```
+{{url-virtualizer}}/open-metadata/admin-services/users/{{user-name}}/servers/{{server-name}}/server-url-root?url={{url-virtualizer}}
+```
+- Configure event bus
+
+**POST** following JSON object 
+````json
+{"consumer": {
+	"bootstrap.servers" :  "{{kafka-server-address}}",
+	"key.deserializer": "org.apache.kafka.common.serialization.StringDeserializer",
+	"value.deserializer": "org.apache.kafka.common.serialization.StringDeserializer"
+},
+"producer": {
+	"bootstrap.servers" :  "{{kafka-server-address}}",
+	"key.serializer": "org.apache.kafka.common.serialization.StringSerializer",
+	"value.serializer": "org.apache.kafka.common.serialization.StringSerializer"
+}
+}
+````
+to the following address
+```
+{{url-virtualizer}}/open-metadata/admin-services/users/{{user-name}}/servers/{{server-name}}/event-bus?connectorProvider=org.odpi.openmetadata.adapters.eventbus.topic.kafka.KafkaOpenMetadataTopicProvider&topicURLRoot={{topic-root}}
+```
+
+- Set up cohort
+
+**POST** to the following address
+```
+{{url-virtualizer}}/open-metadata/admin-services/users/{{user-name}}/servers/{{server-name}}/cohorts/{{cohort-name}}
+```
+
+- Set up virtualizer configuration
+
+**POST** following JSON object 
+````json
+{
+	"class": "VirtualizationConfig",
+	"virtualizationProvider": "GaianDB",
```

Comment:
I would expect a class name for the virtualizationProvider (assuming this is the connector provider for the data virtualization platform's connector).

Need to explain what these values mean so administrator knows how to set them up.

Reply:
Hi Mandy, this is just some dummy information. I will add the items for class name and corresponding properties later.

Reply:
It would be better to have no information than wrong information.